_pages/en_US/troubleshooting.md

```diff
@@ -5,47 +5,17 @@ title: Troubleshooting
 
-### There is no audio or boot splash when launching "LAUNCHER" using Unlaunch
-That is intentional. If you'd like to get those features back, install [hiyaCFW](installing-hiyacfw).
-
-### I get a black screen when trying to launch the Unlaunch installer from TWiLight Menu++
-This is a bug with how the IPC values are cleared (or the lack there of). IPC values are used by audio in TWiLight Menu++.
-
-Until the next release of TWiLight Menu++, go to the TWiLight Menu++ settings, highlight the theme option and then press LEFT/RIGHT until you select the R4 theme. Exit out of TWiLight Menu+++ settings, then launch `UNLAUNCH.DSI`.
-
-Alternatively, you could use an external launcher, such as [GodMode9i](https://github.com/RocketRobz/godmode9i/releases/latest)
-
-## HiyaCFW
-
+## hiyaCFW
 ### "An Error Has Occurred"
-If you get this message when booting hiyaCFW, it means you have too much free space on your SD card. The amount of free space compatible goes by every other range of two gigabytes. For example, 0-2 GB worth of free space works while 2-4 GB doesn't.
-
-Enter the free space on your SD in the box below, press enter, and it will tell you if your SD has a working amount of free space.
-
-<input id="sdSpace" type="number" placeholder="Free space on your SD, in gigabytes (ex. 1.5)" oninput="updateWillWork()">
-Your SD card<span id="willWork">...</span><noscript>is unable to be checked due to JavaScript being disabled. Please enable it in your web browser and try again.</noscript>
-
-If your SD needs less free space, you can easily fill it with dummy files. Listed below are command prompt/terminal commands that will make 1GB worth of dummy files.
- - Windows: `fsutil file createnew dummy0 1073741824`
- - Linux/macOS: `dd if=/dev/zero of=dummy0 count=1024 bs=1048576`
-
-Fill it up until the website says that it will work.
-
-The reason this happens is due to the way the Nintendo DSi System Menu was made. It uses a signed 32-bit integer, which means once an overflow occurs, it will jump to a negative number of free space.
-
-This causes the Nintendo DSi System Menu to freak out, resulting in an "An error has occured" screen. This was never intended by the developers, since the NAND never exceeded 128 MB.
-
-However, by using hiyaCFW, the NAND space is now determined by an exterior factor (your SD card), which can vary in size, resulting in this odd behavior.
+There are several reasons you might get this error:
 
-<script>
-function updateWillWork() {
-  let freeSpace = document.getElementById("sdSpace").value;
-  document.getElementById("willWork").innerHTML = " " + ((freeSpace % 4) < 2 ? "will work!" : "needs dummy files...");
-}
-</script>
+- **Too Much Free Space:** Refollow [Section III of hiyaCFW setup](/hiyacfw-setup#section-iii---maximizing-available-space)
+- **Over 39 Titles:** There is a limit of 39 titles on your DSi System Menu. Delete any additional folders from `/title`
```

Comment:
Also mentioned on Discord, add a bullet for too much (non-system) DSiWare about the 25MB limit

Reply:
"To much DSiWare" uhhh

Reply:
Oh oops lol, do you have an idea for something better than "too much DSiWare"? Its really DSiWare using too much space, but that's kinda long... Otherwise:
```suggestion
- **Over 39 Titles:** There is a limit of 39 titles on your DSi System Menu. Delete any additional folders from `/title`
- **Too much DSiWare:** There is also a limit of 200 blocks (25MB) for DSiWare in the `00030004` folder. This can be worked around by installing as system apps using [TMFH](https://github.com/JeffRuLz/TMFH/releases/latest)
```
There's not a section on TMFH, right? Maybe that should be explained more, but I'll just leave it as this for now

Reply:
Yeah, there isn't a section on TMFH. No one ever made a page for that.
If you'd like to do so, be my guest. I don't own a DSi so I certainly can't.

Reply:
-resolved-